controlplane/rosa/controllers/rosacontrolplane_controller.go

```diff
@@ -327,6 +328,35 @@ func (r *ROSAControlPlaneReconciler) reconcileDelete(ctx context.Context, rosaSc
 	}
 
 	if cluster.Status().State() != cmv1.ClusterStateUninstalling {
+		machinePools, err := rosaScope.GetMachinePools()
+		if err != nil {
+			rosaScope.Info("Error getting controlPlan machinePools ", err)
```

Comment:
why log an error as info? we are returning the error, so it will be logged anyway.